src/embed/index.js

```diff
@@ -0,0 +1,176 @@
+
+container.client.on('paused', pauseOverlay);
+container.client.on('resumed', pauseOverlay);
+container.client.on('loadDone', onLoadComplete);
+container.client.on('loaded', onLoadComplete);
+container.client.on('features', ({ data }) => {
+  if (data.captions) {
+    captionsToggle.removeClass('disabled');
+    captionsToggle.show();
+  }
+  if (data.sound) {
+    soundToggle.removeClass('disabled');
+    soundToggle.show();
+  }
+
+  if (data.soundVolume) {
+    $('#divVolumeRange').removeClass('--disabled');
+  }
+
+  if (data.voVolume) {
+    $('#divVoiceRange').removeClass('--disabled');
+  }
+
+  if (data.musicVolume) {
+    $('#divMusicRange').removeClass('--disabled');
+  }
+
+  if (data.sfxVolume) {
+    $('#divSfxRange').removeClass('--disabled');
+  }
+
+  $('button[data-toggle-div]')['menuToggle']();
+});
+
+container.setupPlugins();
+
+$('form').submit(() => false);
+
+const parseQuery = queryString => {
+  var params = {};
+  var entries = queryString.split('&');
+  for (var i in entries) {
+    var parts = entries[i].split('=');
+    params[decodeURIComponent(parts[0])] = decodeURIComponent(parts[1]);
+  }
+  return params;
+};
+
+const start = () => {
+  // Open via api
+  let api = location.pathname.replace('/embed/', '/api/release/');
+  let playOptions = null;
+  let singlePlay = false;
+  let apiArgs = [];
+  const queryArgs = [];
+
+  if (location.search) {
+    const params = parseQuery(location.search.substr(1));
+
+    // API configurations
+    ['version', 'status', 'commitId', 'token'].forEach(function(param) {
+      if (params[param]) {
+        apiArgs.push(param + '=' + params[param]);
+        delete params[param];
+      }
+    });
+
+    // Check for debug
+    if (params.debug) {
+      apiArgs.push('debug=true');
+      delete params.debug;
+    }
+
+    // Show the controls
+    if (params.controls) {
+      delete params.controls;
+      frame.addClass('show-controls');
+    }
+
+    // Show the title
+    if (params.title) {
+      frame.addClass('show-title');
+
+      delete params.title;
+    }
+
+    // Add the arguments
+    if (apiArgs.length) {
+      api += '?' + apiArgs.join('&');
+    }
+
+    // Single play
+    if (params.singlePlay) {
+      singlePlay = params.singlePlay == 'true' || params.singlePlay == '1';
+      delete params.singlePlay;
+    }
+
+    // Play options
+    if (params.playOptions) {
+      try {
+        playOptions = JSON.parse(params.playOptions);
+      } catch (e) {} // ignore invalid JSON parse
+      delete params.playOptions;
+    }
+
+    // Get any other options and pass them to the query string
+    for (var param in params) {
+      queryArgs.push(param + '=' + params[param]);
+    }
+  }
+
+  container
+    .openRemote(api, {
+      singlePlay,
+      playOptions,
+      query: queryArgs.length ? '?' + queryArgs.join('&') : ''
+    })
+    .catch(err => err.json())
+    .then(({ error }) => alert(error));
```

Comment:
This isn't great, but it's better than the old "Invalid API" message 👎 

Ideally this sort of permissions check could happen at page load in JS (but that's out of scope for this PR)